### ISSUE TITLE
Is Word neccesary ? Images missing in resource folder

### ISSUE THREAD

**Issue by devshank3** [alxnbl/onenote-md-exporter#61]
Hi Developers, the console app is really great...

I ran the console app without MS Word installed....
It worked perfectly.
Even the Intermediate  .docx files where created with images in it.

But in the resource folder the images where missing & not even a single image was present.
PDFs are present in resource folder.

I noticed in the markdowns it referred to this path  "C:\Users\**user**\AppData\Local\Temp\**user**'s Notebook\pandoc\media"
Inside the above path only the images of the last converted page of last section was available.

Some screen shots 
these were my last page in last section. only these were available 
![image](https://user-images.githubusercontent.com/52159502/158961772-f65ab003-d99d-4766-a356-7e477a79176e.png)
![image](https://user-images.githubusercontent.com/52159502/158961685-9b6b56f5-cef2-4999-8c7f-c3ae0aefe003.png)

**Comment by alxnbl** [comment]
Hi @devshank3 , could you export your OneNote notebook please (procedure [here](https://github.com/alxnbl/onenote-md-exporter/blob/main/doc/notebook-onepkg-export.md)), I will try to reproduce.

**Comment by devshank3** [comment]
Yes I'm able to export the whole notebook. 

And I debugged the program with VS2019, The issues I found are

1. The md text returned by `ConvertDocxToMd` in the `ConverterService`  seemed to already have image reference in md format only and it was not a HTML tag.
2. The `ImageAttachements` list seemed to not have any images attached to it. I'm not sure why ........

**Comment by alxnbl** [comment]
Hmm, thanks for the effort to debug. Strange....

Could you try to import the following notebook in ON and try to export it with the tool and see if section swith Image are correctly exported : https://github.com/alxnbl/onenote-md-exporter/blob/main/sample/TestBloc.onepkg . 

If the problem do not appends with the TestBloc notebook, it means there is something specific with the Notebook you are trying to export. In that case, could you join an export of it to this ticket (after removing any personal data).

Thanks a lot

**Comment by devshank3** [comment]
Hi @alxnbl,

Okay... I followed the steps and the result were....

1. I imported TestBloc and exported with your console tool
the images within the tables(Table with Image, Réalisateurs etc)
found their references correctly.... and the images were in the resource folder.

ScreenShots:
![image](https://user-images.githubusercontent.com/52159502/159106871-4dc8cd54-6258-49d4-a60d-4b84bf47a987.png)
![image](https://user-images.githubusercontent.com/52159502/159106883-670b0fdc-959b-48ca-979b-b3709106b708.png)

2. But direct images in the long text, drawings etc did not find the reference to the resource folder & 
the images were not there in the resource folder. They referred to the temp pandoc folder in Appdata/local
ScreenShots:
![image](https://user-images.githubusercontent.com/52159502/159107243-36609e66-4df0-49d7-8263-fc7688131a47.png)
![image](https://user-images.githubusercontent.com/52159502/159107272-812df67d-df3d-4e01-abba-75136b7bc3f0.png)

And the Intermediate word(docx) files contained the images perfectly... I'm not sure were the issue is.... 
ScreenShot:
![image](https://user-images.githubusercontent.com/52159502/159107328-2960934f-b627-4e41-ac9d-0a1988794c8b.png)

And as requested I have attached the export from my OneNote office desktop app, please debug it...
[Debug.zip](https://github.com/alxnbl/onenote-md-exporter/files/8308606/Debug.zip)

**Comment by devshank3** [comment]
And one more observation 

The tables I created in my notebook app in a page and 
the images in the tables are not being referenced correctly  and they are not in the resources folder.

But the tables with images from the testbloc.one seems to work fine. 
The way it is converted in markdown also differs.

example : 
![image](https://user-images.githubusercontent.com/52159502/159110265-b8397283-d441-46e2-a428-de10a0b54862.png)

In this attached .one section note I have included the table I created & the table extracted from testbloc.one. both has images in them. please see their individual behavior. But the images which are directly attached are still not being referred in resources.
 
[Debug2 - Copy.zip](https://github.com/alxnbl/onenote-md-exporter/files/8308728/Debug2.-.Copy.zip)

I partly think there is some issue with my onenote. not sure .......

**Comment by devshank3** [comment]
Okay I kinda figured out the issue, I tried the tool in my other laptop it worked fine.

Mainly I think this issue due to MS Word not installed in my PC.

**But I'm surprised how** `_oneNoteApp.Publish(page.OneNoteId, Path.GetFullPath(docxFileTmpFile), PublishFormat.pfWord);`
**is able to work fine and publish the word file, even if ms word is not installed**.

And here I have attached the results of `mdFileContent ` after `mdFileContent = File.ReadAllText(mdFilePath);`
for both with MS Word & without MS Word installed in my PC.

With MS Word installed:
-
![image](https://user-images.githubusercontent.com/52159502/159114375-2c74f06d-83ba-4514-b0fe-a935e3b071d0.png)
It has HTML img tags whereas the below one doesn't, it is directly converted to md img references

Without MS Word installed:
-
![image](https://user-images.githubusercontent.com/52159502/159114409-626db1c3-449f-4011-bae7-aaed032706d4.png)

I think this is route of the problem. 
Thanks a lot for creating such a great tool.
Especially  I think its great for Joplin users.
